ssc/cmod_pvwattsv7.cpp

```diff
@@ -603,10 +603,12 @@ class cm_pvwattsv7 : public compute_module
 
         // read all the shading input data and calculate the hourly factors for use subsequently
         // timeseries beam shading factors cannot be used with non-annual data
-        auto& table = get_var_table()->lookup("shading")->table;
-        if (table.is_assigned("en_timestep") && table.as_boolean("en_timestep") && !wdprov->annualSimulation())
-//        if (is_assigned("shading:timestep") && !wdprov->annualSimulation())
-            throw exec_error("pvwattsv7", "Timeseries beam shading inputs cannot be used for a simulation period that is not continuous over one or more years.");
+        if (is_assigned("shading")) {
```

Comment:
@sjanzou Need to check here if shading is assigned, or the table.is_assigned("en_timestep") crashes. This fix seems a little clunky because shad.setup also checks if "shading" is assigned, but it works.